
model loaded!
Traceback (most recent call last):
  File "H:\Courses_files\Master\02456_Deep_learning\deepLearningWGAN\Models\DCGAN.py", line 156, in <module>
    DCGAN.evaluate()
  File "H:\Courses_files\Master\02456_Deep_learning\deepLearningWGAN\Models\DCGAN.py", line 152, in evaluate
    utils.save_image(grid, 'Results/img_generatori_iter_{}.png'.format(i))
  File "D:\Users\karl\anaconda3\envs\pytorch2\lib\site-packages\torch\autograd\grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "D:\Users\karl\anaconda3\envs\pytorch2\lib\site-packages\torchvision\utils.py", line 151, in save_image
    im.save(fp, format=format)
  File "D:\Users\karl\anaconda3\envs\pytorch2\lib\site-packages\PIL\Image.py", line 2350, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: 'Results/img_generatori_iter_0.png'